web/src/components/core/InstallButton.tsx

```diff
@@ -81,26 +80,44 @@ according to the provided installation settings.",
  * When clicked, it will ask for a confirmation before triggering the request
  * for starting the installation.
  */
-const InstallButton = (props: Omit<ButtonProps, "onClick">) => {
+const InstallButton = (
+  props: Omit<ButtonProps, "onClick"> & { onClickWithIssues?: () => void },
+) => {
   const issues = useAllIssues();
   const [isOpen, setIsOpen] = useState(false);
   const location = useLocation();
+  const hasIssues = !issues.isEmpty;
 
-  if (!issues.isEmpty) return;
   if (EXCLUDED_FROM.includes(location.pathname)) return;
 
+  const { onClickWithIssues, ...buttonProps } = props;
   const open = async () => setIsOpen(true);
   const close = () => setIsOpen(false);
   const onAccept = () => {
     close();
     startInstallation();
   };
 
+  // TRANSLATORS: The install button label
+  const buttonText = _("Install");
+  // TRANSLATORS: Accessible text included with the install button when there are issues
+  const withIssuesAriaLabel = _("Not possible with current setup. Click to know more.");
```

Comment:
Reviewer, please not that this is part of the install button. Thus, an screen reader will announce "Install not possible with..."

[Kooha-2024-11-25-08-23-04.webm](https://github.com/user-attachments/assets/fdb7c1b1-a59d-4c08-ace5-bd1b113ab9b5)